systems-state-manager/nisystemsstate.yml

```diff
@@ -0,0 +1,999 @@
+            When the inline is true, the
+            Content-Disposition header is set to 'inline'.
+            When the inline is not specified or is
+            false, the state contents are handled as a download. The
+            Content-Disposition header is set to 'attachment' and
+            the MIME type is set to text/x-yaml; charset=UTF-8.
+        type: boolean
+        example: true
+      systemID:
+        description: System ID
+        type: string
+        example: cRIO-9064--SN-11111111--MAC-00-01-02-03-04-05
+    required:
+      - systemID
+
+  RevertStateRequest:
+    type: object
+    title: Revert Request
+    properties:
+      id:
+        description: The ID of the state.
+        type: string
+      version:
+        description: The version of the state.
+        type: string
+    required:
+      - id
+      - version
+    example:
+      id: 5bcde471fc13379595ce9b13
+      version: 9ae65370c79ac158ad061032944afe6b94d927ca
+
+  Error:
+    description: Contains error information
+    type: object
+    properties:
+      name:
+        description: The string error code.
+        type: string
+      code:
+        description: Numeric error code.
+        type: integer
+      resourceType:
+        description: The type of resource associated with the error.
+        type: string
+      resourceId:
+        description: The identifier of the resource associated with the error.
+        type: string
+      message:
+        description: The filled in error message.
+        type: string
+      args:
+        description: Positional argument values for the error code.
+        type: array
+        items:
+          type: string
+      innerErrors:
+        type: array
+        items:
+          $ref: '#/definitions/Error'
+    example:
+      name: Skyline.OneOrMoreErrorsOccurred
+      code: -251040
+      message: >-
+        One or more errors occurred. See the contained list for details of each
+        error.
+      args: []
+      innerErrors:
+        - name: SystemsStateManager.StateNotFound
+          code: -253401
+          resourceType: SystemStateEntry
+          resourceId: 5bc4463af0f06f4585003ae6
+          message: 'The state with identifier 5bc4463af0f06f4585003ae6 was not found.'
+          args: [5bc4463af0f06f4585003ae6]
+
+  Operation:
+    description: An operation provided by the API
+    type: object
+    properties:
+      available:
+        type: boolean
+        description: Whether the operation is available to the caller.
+      version:
+        type: integer
+        description: The version of the available operation.
+    required: [available]
+    example:
+      available: true
+      version: 1
+
+  V1Operations:
+    description: V1 operations
+    type: object
+    properties:
+      operations:
+        description: >-
+          Available operations in the v1 version of the API:
+
+          - getStates: The ability to retrieve the states
```

Comment:
Could we remove the second "the" from this sentence (i.e. "The ability to retrieve states")?